packages/library/src/commonMain/kotlin/io/realm/MutableRealm.kt

```diff
@@ -73,6 +78,39 @@ class MutableRealm : BaseRealm {
         return RealmInterop.realm_is_in_transaction(realmReference.dbPointer)
     }
 
+    /**
+     * Get latest version of an object.
+     *
+     * Realm write transactions always operate on the latest version of data. This method
+     * makes it possible to easily find the latest version of any frozen Realm Object and
+     * return a copy of it that can be modified while inside the write block.
+     *
+     * *Note:* This object is not readable outside the write block unless it has been explicitly
+     * returned from the write.
+     *
+     * @param obj Realm object to look up. Its latest state will be returned. If the object
+     * has been deleted, `null` will be returned.
+     */
+    public fun <T : RealmObject> findLatest(obj: T?): T? {
```

Comment:
I suppose getting a managed object which is not part of the schema will be difficult ;) and since we are already throwing when called with an unmanaged object I would deem it ok for now.

Reply:
But I guess we need to document the `@throws`